tasks/check-data-keys/check-data-keys.yaml

```diff
@@ -24,7 +29,8 @@ spec:
       description: The workspace where the data JSON file resides
   steps:
     - name: check-data-keys
-      image: quay.io/konflux-ci/release-service-utils:e633d51cd41d73e4b3310face21bb980af7a662f
+    # TODO: Update the image to the latest release-service-utils image
+      image: quay.io/redhat-user-workloads/rhtap-release-2-tenant/release-service/release-service-utils:on-pr-403edb0827b86a46933608edc09a8b8598373f0a 
```

Comment:
You have a todo, so hopefully it lasts long enough for you, but fyi the redhat-user-workloads images can vanish at any time without warning, so it is probably better to build a copy of the utils image in your quay org and use that for testing

Reply:
I MR the Utils image today so it's updated now ... I think I need to bump the images elsewhere then.